src/pydrex/core.py

```diff
@@ -628,16 +626,16 @@ def _get_strain_energy(
                 -nucleation_efficiency * dislocation_density ** 2
             )
     elif np.all(rrss == _fabric.RRSS_ENSTATITE):
-        # TODO: What is this? Why is it required?
-        weight_factor = slip_rate_softest * rrss[slip_indices[0]] ** (-stress_exponent)
```

Comment:
Wait, isn't it just `rrss[slip_indices[-1]]`?

Reply:
Yeah, that was the first mistake but then I realised that I haven't seen anyone use a value other than 1 for the rrss on the only active slip system of enstatite. However, I suppose it makes more sense to use `slip_indices` if we're already doing that for olivine.

Reply:
Based on the fortran I think it is `1 / rrss[slip_indices[-1]]` so that's what I've done in the ammended commit

Reply:
Careful, it is either `(1 / rrss[slip_indices[-1]]) ** stress_exponent` or `rrss[slip_indices[-1] ** -stress_exponent`